pkg/core/commands/deployer_create.go

```diff
@@ -282,6 +292,7 @@ and ` + cli.EnvFromFlagName + ` to map values from a ConfigMap or Secret.
 	cmd.Flags().BoolVar(&opts.Tail, cli.StripDash(cli.TailFlagName), false, "watch deployer logs")
 	cmd.Flags().StringVar(&opts.WaitTimeout, cli.StripDash(cli.WaitTimeoutFlagName), "10m", "`duration` to wait for the deployer to become ready when watching logs")
 	cmd.Flags().BoolVar(&opts.DryRun, cli.StripDash(cli.DryRunFlagName), false, "print kubernetes resources to stdout rather than apply them to the cluster, messages normally on stdout will be sent to stderr")
+	cmd.Flags().Int32Var(&opts.ContainerPort, cli.StripDash(cli.ContainerPortFlagName), 0, "`port` to be set as containerPort on the first container of the Deployment")
```

Comment:
I wonder if we should change the default to 8080?

Reply:
then the CLI needs to know the defaults used by the runtime, or should it always set the port to 8080 unless overridden by the user?

Reply:
8080 is the default, but even if it wasn't so what? It's ok for the CLI and the system to have different defaults, so long as the CLI produces valid resources. Ideally they would be the same, or very similar, but it's not critical. Using zero as the default is ok for now, but we may want to rethink it down the road.